src/HonzaBotner.Services/StandUpStreakService.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Threading.Tasks;
+using HonzaBotner.Database;
+using HonzaBotner.Services.Contract;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using StandUpStreak = HonzaBotner.Services.Contract.Dto.StandUpStreak;
+
+namespace HonzaBotner.Services;
+
+public class StandUpStreakService : IStandUpStreakService
+{
+    private readonly HonzaBotnerDbContext _dbContext;
+    private readonly ILogger<WarningService> _logger;
+
+    public StandUpStreakService(HonzaBotnerDbContext dbContext, ILogger<WarningService> logger)
+    {
+        _dbContext = dbContext;
+        _logger = logger;
+    }
+
+    public async Task<StandUpStreak?> GetStreak(ulong userId)
+    {
+        Database.StandUpStreak? standUpStreak = await _dbContext.StandUpStreaks
+            .FirstOrDefaultAsync(streak => streak.UserId == userId);
+
+        return standUpStreak == null ? null : GetDto(standUpStreak);
+    }
+
+    public async Task UpdateStreak(ulong userId)
```

Comment:
There will be an argument about the count of successful tasks.

Reply:
Method only called when I do want to update streak